### PR TITLE
notification listener to run analysis when new data is uploaded

### PR DESCRIPTION
**Pull request by kstoreyf**
- wrote a flask server to listen for object change notifications from a google cloud storage bucket (via app engine proxy), which runs on the gce instance image-analysis using GNU screen
- no longer clean up temp dir in combine_lightcurves.py (breaks with multiple simultaneous uploads, need better solution long term)
- fix hardcoded line in generate_data_network.py

### CODE REVIEW

**Comment by kstoreyf** [comment]
for reasons this pipeline isn't ideal and discussion on next step for master light curve storage, see #4

File: scripts/combine_lightcurves.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import shutil
 import json
 import argparse
-from json.decoder import JSONDecodeError
+#from json.decoder import JSONDecodeError
```
Comment (wtgee) [inline scripts/combine_lightcurves.py]: Can you just remove this commented line and then it looks good to merge. I like that little flask listener, that's snazzy.

**Comment by kstoreyf** [comment]
also note: the app engine proxy code is stored in a cloud source repo: https://pantheon.corp.google.com/code/develop/browse/notification-proxy/master?project=panoptes-survey

not in this repo because it has to be in python27 and i didn't want to mix

**Comment by wtgee** [comment]
Ok. We'll need to see if we need py27 elsewhere. If so, we can assume it will be installed and could specify the name explicitly in the shebang line.

**Pull request merged**